frontend/src/metabase/containers/ActionPicker/ActionOptionItem.tsx

```diff
@@ -1,11 +1,11 @@
 import React from "react";
 
-import { SidebarItem } from "../SidebarItem";
+import { SidebarItem } from "../../dashboard/components/ClickBehaviorSidebar/SidebarItem";
```

Comment:
Can we make `ActionOptionItem` independent from the dashboard code?
I think even duplicating styles would be enough for now

Reply:
I gave it its own set of styles. There's a lot of stuff in SidebarItem we don't need for this.
![Screen Shot 2022-10-04 at 11 34 41 AM](https://user-images.githubusercontent.com/30528226/193887764-b7d898f1-eed0-479a-8ad2-dd7300992392.png)